### PR TITLE
[mxnet][release]Update python version in mx 1.7 containers

### PR DESCRIPTION
**Pull request by SergTogul**
*Issue #, if available:*

## PR Checklist
- [ ] I've prepended PR tag with frameworks/job this applies to : [mxnet, tensorflow, pytorch] | [ei/neuron] | [build] | [test] | [benchmark] | [ec2, ecs, eks, sagemaker]
- [ ] (If applicable) I've documented below the DLC image/dockerfile this relates to
- [ ] (If applicable) I've documented below the tests I've run on the DLC image
- [ ] (If applicable) I've reviewed the licenses of updated and new binaries and their dependencies to make sure all licenses are on the Apache Software Foundation Third Party License Policy Category A or Category B license list.  See [https://www.apache.org/legal/resolved.html](https://www.apache.org/legal/resolved.html).
- [ ] (If applicable) I've scanned the updated and new binaries to make sure they do not have vulnerabilities associated with them.

## Pytest Marker Checklist
- [ ] (If applicable) I have added the marker `@pytest.mark.model("<model-type>")` to the new tests which I have added, to specify the Deep Learning model that is used in the test (use `"N/A"` if the test doesn't use a model)
- [ ] (If applicable) I have added the marker `@pytest.mark.integration("<feature-being-tested>")` to the new tests which I have added, to specify the feature that will be tested
- [ ] (If applicable) I have added the marker `@pytest.mark.multinode(<integer-num-nodes>)` to the new tests which I have added, to specify the number of nodes used on a multi-node test
- [ ] (If applicable) I have added the marker `@pytest.mark.processor(<"cpu"/"gpu"/"eia"/"neuron">)` to the new tests which I have added, if a test is specifically applicable to only one processor type

#### EIA/NEURON Checklist
* When creating a PR:
- [ ] I've modified `src/config/build_config.py` in my PR branch by setting `ENABLE_EI_MODE = True` or `ENABLE_NEURON_MODE = True`
* When PR is reviewed and ready to be merged:
- [ ] I've reverted the code change on the config file mentioned above
#### Benchmark Checklist
* When creating a PR:
- [ ] I've modified `src/config/test_config.py` in my PR branch by setting `ENABLE_BENCHMARK_DEV_MODE = True`
* When PR is reviewed and ready to be merged:
- [ ] I've reverted the code change on the config file mentioned above

## Reviewer Checklist
* For reviewer, before merging, please cross-check:
- [ ] I've verified the code change on the config file mentioned above has already been reverted

*Description:*

*Tests run:*

*DLC image/dockerfile:*

*Additional context:*

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license. I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: test/dlc_tests/conftest.py [hunk 1]
```diff
@@ -326,8 +337,23 @@ def tf_version_within_limit(metafunc_obj, image):
...
     tf24_requirement_failed = "tf24_and_above_only" in metafunc_obj.fixturenames and is_below_tf_version("2.4", image)
+    tf23_requirement_failed = "tf23_and_above_only" in metafunc_obj.fixturenames and is_below_tf_version("2.3", image)
     tf21_requirement_failed = "tf21_and_above_only" in metafunc_obj.fixturenames and is_below_tf_version("2.1", image)
-    if tf2_requirement_failed or tf21_requirement_failed or tf24_requirement_failed:
+    if tf2_requirement_failed or tf21_requirement_failed or tf24_requirement_failed or tf23_requirement_failed:
+        return False
+    return True
+
+
+def mx_version_within_limit(metafunc_obj, image):
```
Comment (saimidu) [inline test/dlc_tests/conftest.py:347]: nit: You could merge this function with the one used for tensorflow, and make it generic.

The `image` variable will already contain the framework name to help identify which `is_below_*_version` function should be used.

File: test/dlc_tests/ec2/test_smclarify.py [hunk 2]
```diff
@@ -16,7 +16,8 @@
 @pytest.mark.integration("smclarify_cpu")
 @pytest.mark.model("N/A")
 @pytest.mark.parametrize("ec2_instance_type", SMCLARIFY_EC2_CPU_INSTANCE_TYPE, indirect=True)
-def test_smclarify_metrics_cpu(training, ec2_connection, region, ec2_instance_type, cpu_only, py3_only):
+@pytest.mark.skip(reason="Skipping for patch branch for MX 1.7 CPU")
+def test_smclarify_metrics_cpu(training, ec2_connection, region, ec2_instance_type, cpu_only, py3_only, tf23_and_above_only, mx18_and_above_only):
     run_smclarify_bias_metrics(training, ec2_connection, region, ec2_instance_type)
```
Comment (saimidu) [inline test/dlc_tests/ec2/test_smclarify.py:23]: Need to give `tf23_and_above_only, mx18_and_above_only` as input to GPU test as well

Comment (SergTogul) [reply test/dlc_tests/ec2/test_smclarify.py:23]: test_smclarify_metrics_gpu works for mxnet 1.7

Comment (saimidu) [reply test/dlc_tests/ec2/test_smclarify.py:22]: It skips because mxnet 1.7 is not cu110

**Review by saimidu — APPROVED** [review]

**Review by saimidu — APPROVED** [review]

**Pull request merged**